src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -135,6 +146,7 @@ public AddressBook load() throws StorageOperationException {
 
         // create empty file if not found
         } catch (FileNotFoundException fnfe) {
```

Comment:
but what happens when executeCommand handles it? We discussed in class that the program will crash. So do think about how your program can handle it more elegantly.